src/Hydrators/ProductHydrator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace RobotStores\Hydrators;
+
+class ProductHydrator
+{
+    private array $productsArray;
+
+    static public function getFilteredProducts(int $characterId = null):Array {
```

Comment:
perhaps remove the parameter $ character id altogether here, as it's not part of this story?
also space before return type

Reply:
return type is capitalised here and nowhere else